### PR TITLE
allow ccplot to be empty when plotting cc curve

### PR DESCRIPTION
**Pull request by robynstuart**
Users should not have to specify a value for the upper limit of the x axis in order to create & display the cost coverage curve. Note that this should be fixed with the BE changes here, should not require FE changes.

### CODE REVIEW

File: server/src/sim/makeccocs.py [hunk 1]
```diff
@@ -25,8 +25,8 @@
 default_init_ccparams = []
 default_init_convertedccparams = []
 default_init_nonhivdalys = [0.0]
-default_makeplot = 0 # CK: Otherwise brings up >100 figures
-default_effect = [['sex', 'condomcas'], [u'MSM']] # D.programs[default_progname]['effects'][0]
```
Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: do you want to have figures by default?

Comment (robynstuart) [inline server/src/sim/makeccocs.py]: no

Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: ok, let's change it back to 0 then :)

File: server/src/sim/makeccocs.py [hunk 2]
```diff
@@ -25,8 +25,8 @@
...
+default_makeplot = 1 # CK: Otherwise brings up >100 figures
```
Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: This breaks web environment, because it references to D which is not in scope.

Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: Such things are better to define within the method, when you have D available.

Comment (robynstuart) [inline server/src/sim/makeccocs.py]: sorry anna... i just changed it to test something and forgot to change it back

Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: please commit the right version to the same branch :)

**Comment by Ashalynd** [comment]
@dmitryevseev as you are working on the same part now, please review the FE changes here

**Pull request merged**